src/maingoblint.ml

```diff
@@ -10,8 +10,11 @@ let writeconffile = ref ""
 
 (** Print version and bail. *)
 let print_version ch =
-  let open Version in
-  printf "Goblint version: %%VERSION_NUM%% (%s)\n" goblint;
+  let release_version = "%%VERSION_NUM%%" in
+  let version =
+    if String.starts_with release_version "%" then ConfigVersion.version
+    else release_version in
```

Comment:
This is the exact kind of logic that should go into `version.ml` such that our version display is consistent across all four places.